From 1d5c651321a87e218995821b0d148867db977988 Mon Sep 17 00:00:00 2001
From: Mark Washenberger <mark.washenberger@rackspace.com>
Date: Tue, 25 Sep 2012 19:57:10 +0000
Subject: [PATCH] Pass empty args to test config parser.

This fixes bug 1056420, which allows users to pass nosetests options
through run_tests.sh again.

Change-Id: I03e84488a24e2552b61aa0cab842d8325e8e856f
(cherry picked from commit ef641597dbfc0fcb856c143b213be493ab44cc65)
---
 glance/tests/unit/test_clients.py |    2 +-
 glance/tests/utils.py             |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/glance/tests/unit/test_clients.py b/glance/tests/unit/test_clients.py
index 20200b3..5be190d 100644
--- a/glance/tests/unit/test_clients.py
+++ b/glance/tests/unit/test_clients.py
@@ -39,7 +39,7 @@ UUID1 = _gen_uuid()
 UUID2 = _gen_uuid()
 
 #NOTE(bcwaldon): needed to init config_dir cli opt
-config.parse_args()
+config.parse_args(args=[])
 
 
 class TestBadClients(test_utils.BaseTestCase):
diff --git a/glance/tests/utils.py b/glance/tests/utils.py
index 370fc8b..cb9278d 100644
--- a/glance/tests/utils.py
+++ b/glance/tests/utils.py
@@ -57,7 +57,7 @@ class BaseTestCase(unittest.TestCase):
         #NOTE(bcwaldon): parse_args has to be called to register certain
         # command-line options - specifically we need config_dir for
         # the following policy tests
-        config.parse_args()
+        config.parse_args(args=[])
 
     def tearDown(self):
         super(BaseTestCase, self).tearDown()
